addons/account/data/mail_template_data.xml

```diff
@@ -264,104 +264,177 @@
     <t t-out="company_name"/>
 </p>
         </template>
-       <record id="mail_template_einvoice_notification" model="mail.template">
-            <field name="name">New eInvoices Notification</field>
-            <field name="subject">New Electronic Invoices Received</field>
-            <field name="email_from">{{ object.company_id.email_formatted }}</field>
-            <field name="email_to">{{ object.incoming_einvoice_notification_email }}</field>
+
+        <!-- Journal notifications -->
+        <record id="mail_template_invoice_subscriber" model="mail.template">
+            <field name="name">Journal Notification</field>
+            <field name="model_id" ref="account.model_account_move"/>
+            <field name="subject">{{ object.company_id.name }} - New item in {{ object.journal_id.display_name or 'Invoices' }} journal</field>
+            <field name="email_from">{{ (object.company_id.email_formatted or user.email_formatted) }}</field>
             <field name="use_default_to" eval="False"/>
-            <field name="model_id" ref="account.model_account_journal"/>
             <field name="auto_delete" eval="True"/>
-            <field name="description">Notification email for newly received eInvoices</field>
             <field name="body_html" type="html">
+<t t-set="company" t-value="object.company_id"/>
+<t t-set="partner" t-value="object.partner_id"/>
+<t t-set="is_sale" t-value="object.is_sale_document(include_receipts=True)"/>
+<t t-set="is_purchase" t-value="object.is_purchase_document(include_receipts=True)"/>
+<t t-set="document_label" t-value="object._get_move_display_name()"/>
+<t t-set="total_amount" t-value="format_amount(object.amount_total, object.currency_id or object.company_currency_id) if object.amount_total else False"/>
+<t t-set="portal_url" t-value="object.get_portal_url()"/>
+<t t-set="unsubscribe_url" t-value="ctx.get('unsubscribe_url', False)"/>
+
+<t t-set="has_partner" t-value="bool(partner)"/>
+<t t-set="has_invoice_date" t-value="bool(object.invoice_date)"/>
+<t t-set="has_due_date" t-value="bool(object.invoice_date_due)"/>
+<t t-set="has_payment_state" t-value="bool(object.payment_state and object.payment_state != 'not_paid')"/>
```

Comment:
The payment state is always set. Also
```suggestion
<t t-set="has_payment_state" t-value="bool(object.payment_state != 'not_paid')"/>
```
Also those additional booleans are useless imo. What about:
```suggestion
<t t-set="payment_state" t-value="object.payment_state != 'not_paid' and object.payment_state.replace('_', ' ')"/>
```
Then, to check the payment_state, you just need `if payment_state`